qtconsole/console_widget.py

```diff
@@ -494,6 +494,13 @@ def sizeHint(self):
         Outputs are not displayed until enter is pressed.
         """
     )
+
+    other_output_prefix = Unicode('[remote] ', config=True,
```

Comment:
That's the default used in the jupyter console client. I copied the feature from there.